codebasin/__main__.py

```diff
@@ -243,5 +257,5 @@ def report_enabled(name):
         sys.argv[0] = "codebasin"
         main()
     except Exception as e:
-        logging.getLogger("codebasin").error(str(e))
+        log.error(str(e))
```

Comment:
Is this purposefully done to not include the traceback? Otherwise you can do `log.exception(e)`

Reply:
Yes, exactly.  When `codebasin` is run as a user-facing application, we want to catch all the exceptions and not reveal any traceback information.